packages/client/src/components/maps/MapSlideUpSheet/MapSlideUpSheet.js

```diff
@@ -9,54 +9,66 @@ import SlidingUpPanel from 'rn-sliding-up-panel'
 const { height } = Dimensions.get('window')
 
 class MapSlideUpSheet extends React.Component {
+
   render() {
+
     return (
       <React.Fragment>
         <SlidingUpPanel
           ref={(c) => (this._panel = c)}
           draggableRange={{ top: height / 1.15, bottom: 120 }}
           animatedValue={this._draggedValue}
           showBackdrop={true}
+          panelStyle={styles.panel}
         >
-          <View style={styles.panel}>
-            <TouchableOpacity onPress={() => this._panel.show()}>
-              <View aria-label='SearchPanel' style={styles.panelHeader}></View>
-            </TouchableOpacity>
-            <View style={styles.container}>
-              <SearchBar />
-              <ResultSorter />
-              <SearchView navigation={this.props.navigationProp} />
+          {dragHandler  => (
+            <View style={styles.panel}>
+              <View {...dragHandler}>
+                <TouchableOpacity onPress={() => this._panel.show()}>
+                  <View aria-label='SearchPanel' style={styles.panelHandle}></View>
+                </TouchableOpacity>
+                <SearchBar />
+                <ResultSorter buttonTextStyle={styles.buttonStyle}/>
+              </View>
+              <View style={styles.listContainer}>
+                <SearchView navigation={this.props.navigationProp} />
+              </View>
             </View>
-          </View>
+          )}
         </SlidingUpPanel>
       </React.Fragment>
     )
   }
 }
 
 export const styles = StyleSheet.create({
-  container: {
+
+  listContainer: {
     alignItems: 'center',
     backgroundColor: '#f8f9fa',
     flex: 1,
     justifyContent: 'flex-start',
-    marginTop: 15,
   },
   panel: {
-    backgroundColor: 'white',
+    backgroundColor: '#f8f9fa',
     borderRadius: 15,
     flex: 1,
     position: 'relative',
   },
-  panelHeader: {
+  panelHandle: {
     alignSelf: 'center',
     backgroundColor: '#E7E7E7',
     borderRadius: 5,
     height: 5,
     justifyContent: 'center',
+    marginBottom: 20,
     marginTop: 10,
     width: '36%',
   },
+  buttonStyle: {
```

Comment:
This style isn't used. Do we need it?

Reply:
My mistake, was testing something and forgot to remove it.